tools/config/BUILD

```diff
@@ -35,14 +57,14 @@ config_setting(
 config_setting(
     name = "webasm",
     values = {
-        "cpu": "webasm",
+        "define": "webassembly=true",
```

Comment:
Reverting this bit enables me (in conjunction with reverting the below bit in `webasm_opt`) to get a successful build.

Reply:
That indeed does the trick. Not sure why I was running into issues earlier.